sqldelight-gradle-plugin/src/test/kotlin/com/squareup/sqldelight/PluginTest.kt

```diff
@@ -57,5 +57,11 @@ class PluginTest {
         .withArguments("clean", "generateSqlDelightInterface", "--stacktrace")
         .build()
     assertThat(result.output).contains("BUILD SUCCESSFUL")
+
+    // Assert the plugin added the common dependency
+    val dependenciesResult = runner
+        .withArguments("dependencies", "--stacktrace")
```

Comment:
i wanted to just run an actual build to verify things are fine but it was consistently failing and im not sure why